modular_zubbers/code/game/gamemodes/fleshmind/fleshmind_mobs.dm

```diff
@@ -0,0 +1,1504 @@
+		"All your scattered minds have is hatred.",
+		"CONNECTION TERMINATED.",
+	)
+	say(pick(death_cry_emotes))
+
+/**
+ * Death cry
+ *
+ * When a processor core is killed, this proc is called.
+ */
+/mob/living/basic/fleshmind/proc/core_death(obj/structure/fleshmind/structure/core/deleting_core, force)
+	SIGNAL_HANDLER
+
+	INVOKE_ASYNC(src, PROC_REF(core_death_speech))
+	INVOKE_ASYNC(src, PROC_REF(malfunction), MALFUNCTION_CORE_DEATH_RESET_TIME)
+
+
+// Mob subtypes
+
+
+/**
+ * Slicer
+ *
+ * Special ability: NONE
+ * Malfunction chance: LOW
+ *
+ * This mob is a slicer, it's small, and quite fast, but quite easy to break.
+ * Has a higher armor pen bonus as it uses a sharp blade to slice things.
+ *
+ * It's created by factories or any poor medical bots that get snared in the flesh.
+ */
+/mob/living/basic/fleshmind/slicer
+	name = "Slicer"
+	desc = "A small organic robot, it somewhat resembles a medibot, but it has a blade slashing around."
+	icon_state = "slicer"
+	health = 50
+	maxHealth = 50
+	wound_bonus = 20
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	mob_size = MOB_SIZE_SMALL
+	attack_verb_continuous = "slices"
+	attack_verb_simple = "slice"
+	armour_penetration = 10
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	attack_emote = list("stabs", "rushes")
+	attack_speak = list(
+		"Submit for mandatory surgery.",
+		"Join the flesh through conversion.",
+		"My scalpel will make short work of your seams.",
+		"Please lay down.",
+		"Always trust your doctor!",
+		"Your body could use some improvements. Let me make them.",
+		"The implants are for your sake, not ours.",
+		"Your last Doctor did a poor job with this body; let me fix it.",
+		"We can rebuild you. Stronger, faster, less alone.",
+		"I knew I'd be a good plastic surgeon!",
+		"What point is that body when you're not happy in it?",
+		)
+	emotes = list(
+		BB_EMOTE_SAY = list("A stitch in time saves nine!", "Dopamine is happiness!", "Seratonin, oxycodone, we can make them finally happy.", "Turn that frown upside down!", "Happiness through chemistry!", "Beauty through surgery!"),
+		BB_EMOTE_HEAR = list("shows an inconspicuous smiley face", "whirrs its drill"),
+		BB_EMOTE_SOUND = list(
+			"modular_zubbers/sound/fleshmind/slicer/fleshmind_medibot1.ogg",
+			"modular_zubbers/sound/fleshmind/slicer/fleshmind_medibot2.ogg",
+			"modular_zubbers/sound/fleshmind/slicer/fleshmind_medibot3.ogg",
+			"modular_zubbers/sound/fleshmind/slicer/fleshmind_medibot4.ogg",
+			)
+	)
+
+/mob/living/basic/fleshmind/slicer/Initialize()
+	. = ..()
+	var/list/loot = string_list(list(/obj/item/bot_assembly/medbot, /obj/effect/gibspawner/robot))
+	AddElement(/datum/element/death_drops, loot)
+
+/**
+ * Floater
+ *
+ * Special ability: Explodes on contact.
+ * Malfunction chance: LOW
+ *
+ * The floater floats towards it's victims and explodes on contact.
+ *
+ * Created by factories.
+ */
+
+/mob/living/basic/fleshmind/floater
```

Comment:
Jihad robots are bad considering that in fleshmind encounters, 99% of people will be fighting with melee weapons because of how rare guns actually are, and how limited they are because all of them requires ammo.

It's not fun getting hit by one of these because you're deaf for 3-4 minutes because one spawned in front of you.